pycbc/distributions/sky_location.py

```diff
@@ -46,6 +46,87 @@ class UniformSky(angular.UniformSolidAngle):
     _default_azimuthal_angle = 'ra'
 
 
+class UniformCircleSky:
+
```

Comment:
Add a docstring to this class to explain what it does. Use the docstring of `FisherSky` as a model.